### PR TITLE
[update] update dashboard response

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?
update dashboard response

### What is changed and how does it work?
Modify the API error return structure.
Handle the cases where there is no return value for POST request and PUT request.

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
Is there any other API has such issue?

**Comment by fewdan** [comment]
> Is there any other API has such issue?

Previously, there was no return value after the post request and the put request were successful, so these two cases were modified.

File: pkg/apiserver/experiment/experiment.go [hunk 1]
```diff
@@ -110,12 +110,17 @@ type Detail struct {
 type createExperimentFunc func(*core.ExperimentInfo) error
 type updateExperimentFunc func(*core.ExperimentYAMLDescription) error
 
+// StringStruct defines a common status struct.
+type StringStruct struct {
```
Comment (g1eny0ung) [inline pkg/apiserver/experiment/experiment.go:114]: How about naming it `StatusResponse`?

**Review by g1eny0ung — APPROVED** [review]
LGTM 👍🏽

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**